analyses/cell-type-wilms-tumor-06/00_run_workflow.R

```diff
@@ -35,6 +35,7 @@ metadata <- read.table(sample_metadata_file, sep = "\t", header = TRUE)
 module_base <- file.path(root_dir, "analyses", "cell-type-wilms-tumor-06")
 
 # Download and create the references for label transfer, and download the homolog file for ID mapping ----------
+system(command = glue::glue("Rscript ", file.path(module_base, "scripts", "download-reference-files.R")))
```

Comment:
If you do the downloads here rather than in a separate script, you can include the file paths as the arguments to the later commands, making any later changes easier. 

```suggestion
# The fetal kidney atlas (Stewart) RDS file from CELLxGENE.
kidney_url <- "https://datasets.cellxgene.cziscience.com/40ebb8e4-1a25-4a33-b8ff-02d1156e4e9b.rds"
kidney_file <- file.path(root_dir, "scratch", "fetal_kidney.rds")
if (!file.exists(kidney_file)) {
  download.file(url = kidney_url, destfile = kidney_file)
}

# The homologs.rds file from Azimuth
homologs_url <- "https://seurat.nygenome.org/azimuth/references/homologs.rds"
homologs_file <- file.path( root_dir, "scratch", "homologs.rds")
if (!file.exists(homologs_file)) {
  download.file(url = homologs_url, destfile = homologs_file)
}
```